### DISCUSSION TITLE
Random field generated for a slope model.

### DISCUSSION THREAD

**Discussion by LanPeng-94** [GeoStat-Framework/GSTools#335] [Q&A]
Hi,

I just started working with gstools, and am not good at it. Please give me some suggestions. 

I want to generate the random field for the following slope model.

![image](https://github.com/GeoStat-Framework/GSTools/assets/52651366/3ba76338-a37d-44d7-bccc-d53506cb7ca8)

I can obtain the coordinates of all grid nodes. For this model, what should I do using gstools to obtain the random field? 

Regards,
Peng

**Comment by LSchueler** [comment]
Hi Peng,

what exactly are the problems you encounter? Did you have a look at the tutorials we provide? These are [two](https://geostat-framework.readthedocs.io/projects/gstools/en/latest/examples/01_random_field/05_mesh_ensemble.html#sphx-glr-examples-01-random-field-05-mesh-ensemble-py) [examples](https://geostat-framework.readthedocs.io/projects/gstools/en/latest/examples/01_random_field/06_pyvista_support.html) which use external meshes, but if you have your mesh as numpy arrays, it should be pretty straight forward and you can have a look at some of the more simpler examples.
